src/templates/build-debug/build-debug-android.ejf

```diff
@@ -0,0 +1,81 @@
+<%
+const isMonorepo = props.pathRelativeToRoot !== '.'
+
+const install = {
+  yarn: "yarn install --immutable",
+  npm: "npm ci"
+}
+
+const build = {
+  yarn: `yarn ${(isMonorepo) ? `--cwd ${props.pathRelativeToRoot} ` : ""}build:debug:android`,
+  npm: `npm run build:debug:android ${(isMonorepo) ? `--prefix ${props.pathRelativeToRoot}` : ""}`
+}
+%>
+
+name: Build Android Debug App
+
+on:
+  workflow_call:
+    outputs:
+      build-cache-key:
+        description: 'The primary key to used for caching the build'
```

Comment:
without `to` ?

Reply:
thanks, changed in ef4d52054da13215e5dbe33ba57e407e5fe26939